### PR TITLE
THE melee update

### PR DESCRIPTION
**Pull request by CUST-0M**
## About The Pull Request

Big melee update.
-Buffs big leagues by 20%
-Nerfs some power fists.
-Some small balance changes to make underutilized melee weapons more useful.
-Gives Vexillarius and Legion Veterans access to the new melee belt
-Auxilia, Forgemasters and anyone that reads the grognak book can craft the melee belt.

There's also a new feature in the form of a belt which gives you run speed and a leg sweep in exchange for being unable to use guns. It takes a while to put on to prevent abuse, so it's basically an option to go faster and have a leg sweep in exchange for your ability to use guns if you want to focus on that melee combat grindset. It can only be worn by people with big leagues cause why the hell would you use it without it anyway. Is it going to be good or just pure garbage? Only one way to find out.

## Why It's Good For The Game

Generally makes melee better, big leagues could use a buff and instead of just raising numbers there's a new feature being added, so there'll be something new for people to engage with and try out.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
add: melee combat belt
add: melee combat belt on the melee loadouts for veteran and vex
add: forgemaster, aux and people who read grognak's can craft the melee belt
tweak: minor melee weapon value changes
tweak: big leagues buffed by 20%
/:cl:

### CODE REVIEW

**Comment by 5467543** [comment]
nerf two range shit other than the standard spear, standard spear is fine
the big leagues un-nerf was necessary (lone star coders salt PR'd it out)
the claw spear nerf is good
some of the armor pen values are insane (THE LOUISVILLE HAS .65 AP) so maybe retune them, really .3 should be the max

File: code/game/objects/items/melee/f13twohanded.dm [hunk 1]
```diff
@@ -33,13 +33,14 @@
...
-// Legion Axe		Keywords: Damage 30/60
+// Legion Axe		Keywords: Damage 30/50 AP 0.25
 /obj/item/twohanded/legionaxe
 	name = "War Honed Axe"
 	desc = "Heavy fireman axe from the old world, Restored to working order by legion craftsmen. Excellent for smashing doors or heads."
 	icon_state = "legionaxe"
 	icon_prefix = "legionaxe"
-	force = 30
+	force = 25
+	armour_penetration = 0.25
```
Comment (chemistrymain2) [inline code/game/objects/items/melee/f13twohanded.dm:43]: No, keep the axe as is without AP

File: code/game/objects/items/melee/f13twohanded.dm [hunk 2]
```diff
@@ -80,7 +81,8 @@
 	desc = "Heavy fireman axe from the old world, with its distinctive red colour and excellent quality steel. Excellent for smashing doors."
 	icon_state = "fireaxe"
 	icon_prefix = "fireaxe"
-	force = 26
+	force = 21
+	armour_penetration = 0.15
```
Comment (chemistrymain2) [inline code/game/objects/items/melee/f13twohanded.dm:85]: Ditto

File: code/game/objects/items/melee/f13twohanded.dm [hunk 3]
```diff
@@ -362,38 +364,29 @@
 	desc = "There ain't a skull in the league that can withstand a swatter, especially with large nails drilled through the top of it."
 	icon_state = "baseballspike"
 	icon_prefix = "baseballspike"
-	force = 26
+	force = 20
+	armour_penetration = 0.40
```
Comment (chemistrymain2) [inline code/game/objects/items/melee/f13twohanded.dm:368]: Absolutely fucking not

File: code/game/objects/items/melee/f13twohanded.dm [hunk 4]
```diff
@@ -362,38 +364,29 @@
...
 
-// Louisville Slugger		Keywords: Damage 25/32, Damage bonus Stamina
+// Louisville Slugger		Keywords: Damage 23/27, AP 0.65 Damage bonus Stamina
 /obj/item/twohanded/baseball/louisville
 	name = "Louisville slugger"
 	desc = "Purification in progress."
 	icon_state = "louisville"
 	icon_prefix = "louisville"
+	force = 23
+	armour_penetration = 0.65
```
Comment (chemistrymain2) [inline code/game/objects/items/melee/f13twohanded.dm:384]: No

File: code/game/objects/items/melee/f13twohanded.dm [hunk 5]
```diff
@@ -465,7 +458,7 @@
 	inhand_y_dimension = 64
 	damtype = "fire"
 	force = 5
-	armour_penetration = 0.1
+	armour_penetration = 0.3
```
Comment (chemistrymain2) [inline code/game/objects/items/melee/f13twohanded.dm:461]: This is fine

File: code/modules/jobs/job_types/legion.dm [hunk 6]
```diff
@@ -704,7 +705,7 @@ commented out pending rework*/
 	suit_store = /obj/item/gun/ballistic/automatic/smg/mp5
 	glasses = /obj/item/clothing/glasses/sunglasses/big
 	backpack_contents = list(
-		/obj/item/ammo_box/magazine/uzim9mm = 2,
+		/obj/item/melee/powered/ripper = 1,
```
Comment (chemistrymain2) [inline code/modules/jobs/job_types/legion.dm:708]: No, no ripper for explorers on their best loadout

Give them a gladius if you want

**Comment by chemistrymain2** [comment]
All of those changes I pointed out are problematic

Also big leagues being multiplied by .45 n top of all these changes would make melee incredibly lethal

Either take the .45 multiplier, or these changes, not both

AP = damage, so remember that by adding AP to a weapon, you are massively increasing its damage

**Comment by chemistrymain2** [comment]
Okay. Explorer ripper loadout only is really funny.

Fuck it go ahead.

**Comment by CUST-0M** [comment]
reverted all the two handed stuff except the axe which i toned down because i wanna see veterans frag ncr conscripts with it.

**Comment by ogrigbe** [comment]
This will be most foul.

**Comment by 5467543** [comment]
> All of those changes I pointed out are problematic
> 
> Also big leagues being multiplied by .45 n top of all these changes would make melee incredibly lethal
> 
> Either take the .45 multiplier, or these changes, not both
> 
> AP = damage, so remember that by adding AP to a weapon, you are massively increasing its damage

(protip take the bigleagues change)

**Comment by Weaselburg** [comment]
x .45 damage will make a 20 melee weapon go to 29, 30 go to 45, 50 go to 75, etc. That's INCREDIBLY lethal in a meta that already heavily favors melee, and is in essence just a legion buff since they're the only faction that gets it en masse and you can't really get it anywhere other then spawning with it now.

The other stuff is fine.

**Comment by 5467543** [comment]
> 

it literally was like that for years before it was nerfed, the .45 bigleagues is good but giving shit armorpen is way way better than a damage increase

**Comment by Weaselburg** [comment]
> it literally was like that for years before it was nerfed after people used melee for a single month, the .45 bigleagues is good but giving shit armorpen is way way better than a damage increase
frok already make that obvious. I didn't feel like reiterating their point.

**Comment by ogrigbe** [comment]
broken forever

**Comment by AllCorruptingWyrm** [comment]
worse comes to worst, buffout should be the gateway to the new and improved big leagues 
but genuinely love these changes here because melee is fun.
 
image

**Comment by ogrigbe** [comment]
Are you going to fix itttttt

**Comment by chemistrymain2** [comment]
Do you want me to just add the belt dude

**Comment by ogrigbe** [comment]
going to close it because broken forever

**Pull request closed without merging**